source/Halibut.Tests/Timeouts/ReceiveResponseTimeoutTests.cs

```diff
@@ -110,8 +110,9 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
                 var returnDataStreamService = clientAndService.CreateAsyncClient<IReturnSomeDataStreamService, IAsyncClientReturnSomeDataStreamService>();
 
                 // Act
-                (await AssertionExtensions.Should(() => returnDataStreamService.SomeDataStreamAsync()).ThrowAsync<HalibutClientException>())
-                    .And.Message.Should().ContainAny(
+                var e = (await AssertionExtensions.Should(() => returnDataStreamService.SomeDataStreamAsync()).ThrowAsync<HalibutClientException>()).And;
+                Logger.Information(e, "The received expected exception, we were expecting one");
```

Comment:
I saw this fail locally and didn't have the details I needed to find out why, so log the exception